### PR TITLE
Input

### PR DESCRIPTION
**Pull request by JantuDeb**
**Added different variants of inputs**

- Default inputs.
- Error inputs for validation.
- Search input, input with an action icon and text.
- Floated label input

[Preview](https://deploy-preview-14--flamingoui.netlify.app/documentation/html/input)

### CODE REVIEW

**Review by munjalSaksham71 — COMMENTED** [review]
Code is clean, just suggestion: can you make the error red font size to little small and the margin between them should be less...

Otjer than this, it is good to merge

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+.error:focus {
+  outline: 3px solid var(--fu-blue-light);
+  border: 1px solid transparent;
+}
+
+
+
+/* INPUT WITH */
+.input-icon {
```
Comment (ShreyashMahajan) [inline components/input.css:34]: you can use px for less than 1rem

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  outline: 2px solid var(--fu-blue-light);
+  border: none;
+}
+
+.input-action span {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--fu-gray-300);
+  padding: 0 0.5rem;
```
Comment (ShreyashMahajan) [inline components/input.css:79]: same here, you can use px for less than 1rem

File: components/input.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+/* FLOATED INPUT */
+
+.floated{
+  position: relative;
+  color: var(--fu-gray-600);
+}
+
+.input-floated{
+  padding: 0.2rem 0;
```
Comment (ShreyashMahajan) [inline components/input.css:93]: same as above

**Review by ShreyashMahajan — APPROVED** [review]
overall code looks very clean. One thing which can be changed is that you can use px for less than 1rem (just a suggestion).

File: components/input.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  border: none;
+}
+
+.input-action input:focus {
+  outline: 2px solid var(--fu-blue-light);
+  border: none;
+}
+
+.input-action span {
+  display: flex;
```
Comment (Manju-sree) [inline components/input.css:75]: avoid using flex property repetitive in code
input-action span {
  display: flex;} and  
.input-action {
  display: flex;

**Review by Manju-sree — APPROVED** [review]

File: components/input.css (hunk 4 above)
Comment (JantuDeb) [reply components/input.css:75]: This actually applies only to span element.

**Pull request merged**